pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/search/ElasticSearch.java

```diff
@@ -187,12 +187,22 @@ public ElasticSearch getParticipantsWithinRange(String esParticipantsIndex, int
 
     @Override
     public ElasticSearch getParticipantsByIds(String esIndex, List<String> participantIds) {
+        BoolQueryBuilder boolQuery = getBoolQueryOfParticipantsId(participantIds);
+        return getElasticSearchParticipants(esIndex, participantIds, boolQuery);
+    }
+
+    public ElasticSearch getParticipantsByShortIds(String esIndex, List<String> participantShortIds) {
+        BoolQueryBuilder boolQuery = getBoolQueryOfParticipantsShortId(participantShortIds);
+        return getElasticSearchParticipants(esIndex, participantShortIds, boolQuery);
+    }
+
+    private ElasticSearch getElasticSearchParticipants(String esIndex, List<String> participantIds, BoolQueryBuilder boolQuery) {
```

Comment:
onc history upload is by shortId (hruid) , hence this refactor to reuse existing code by guid lookup